File: UI/public/app/feeds/feed.js
var app;
(function (app) {
    var domain;
    (function (domain) {
        var FeedItem = (function () {
            function FeedItem(link, title, description, uid) {
                this.link = link;
                this.title = title;
                this.description = description;
                this.uid = uid;
            }
            return FeedItem;
        })();
        domain.FeedItem = FeedItem;
        var Feed = (function () {
            function Feed(feedId, name, feedUrl, feedItems) {
                this.feedId = feedId;
                this.name = name;
                this.feedUrl = feedUrl;
                this.feedItems = feedItems;
            }
            return Feed;
        })();
        domain.Feed = Feed;
    })(domain = app.domain || (app.domain = {}));
})(app || (app = {}));

File: FeedrInfrastructure/Model/Feed.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace FeedrInfrastructure.Model
{
    public class Feed
    {
        /// <summary>
        /// Unique FeedId
        /// </summary>
        public string feedId { get; set; }

        /// <summary>
        /// Name of the site
        /// </summary>
        public string name { get; set; }

        /// <summary>
        /// Default feed url. Feed url should reflect the pagination index too.
        /// </summary>
        public string feedUrl { get; set; }

        /// <summary>
        /// Collection of <see cref="FeedItem"/>
        /// </summary>
        public List<FeedItem> feedItems { get; set; }
    }
}
